### PR TITLE
seker kod bootstrap

### PR DESCRIPTION
**Pull request by fedidat**

### CODE REVIEW

File: signin.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                <div class="navbar-nav ml-auto">
+                    <a class="nav-link" href="users.html">Users</a>
+                    <a class="nav-link" href="signin.html">Sign in</a>
+                    <a class="nav-link" href="signup.html">Sign up</a>
+                </div>
+            </div>
+        </nav>
+
+        <main class="container" role="main" >
+            <form class="col-md-6 offset-md-3">
```
Comment (KarinaBuk96) [inline signin.html]: "col"s should be wrapped with"row"s, which should be wrapped with "container"
and you should also use "col-lg-..." , "col-sm-... " classes and etc. to support windows and screens of different sizes

File: signup.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                <div class="navbar-nav ml-auto">
+                    <a class="nav-link" href="users.html">Users</a>
+                    <a class="nav-link" href="signin.html">Sign in</a>
+                    <a class="nav-link" href="signup.html">Sign up</a>
+                </div>
+            </div>
+        </nav>
+
+        <main class="container" role="main" >
+            <form class="col-md-6 offset-md-3">
```
Comment (KarinaBuk96) [inline signup.html]: "col" should be inside a "row"

File: signup.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                <fieldset class="group-border">
+                    <legend class="group-border">Sign up</legend>
+                    <div class="form-group row">
+                        <label class="col-4 col-form-label">Username</label>
+                        <div class="col-8">
+                            <input class="form-control" type="text"  name="username">
+                        </div>
+                    </div>
+                    <div class="form-group row">
+                        <label class="col-4 col-form-label">Password</label>
```
Comment (KarinaBuk96) [inline signup.html]: you can add the class "input-group-addon" to the input label/span inside the "form-group"

File: css/tweets.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
+textarea {
+    width: 100%;
+}
+
+form, .tweets {
+    margin-bottom: 100px;
```
Comment (KarinaBuk96) [inline css/tweets.css]: you can use bootstrap instead of margins

File: css/tweets.css [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+.tweets img {
+    height: 80px;
```
Comment (KarinaBuk96) [inline css/tweets.css]: use "%" instead of "px" where possible

File: css/users.css [hunk 6]
```diff
@@ -0,0 +1,20 @@
+form {
+    margin: 20px;
+}
+
+.navbar .navbar-brand > img { 
+    max-height: 40px; 
+}
+
+.userlist * {
+    text-align: center;
```
Comment (KarinaBuk96) [inline css/users.css]: you can use bootstrap instead

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+.userlist img {
+    height: 120px;
+}
+
+
+
+.row{
+    margin-top: 10px;
+    margin-bottom: 10px
+  }
```
Comment (KarinaBuk96) [inline css/style.css]: for most things here - you can use bootstrap instead
but if you implement something by yourself - use "%", not "px"

**Review by KarinaBuk96 — APPROVED** [review]

File: css/tweets.css (hunk 5 above)
Comment (fedidat) [reply css/tweets.css]: Disagree. For images, until I have a server side rescaling enforcing unified image size (and even then...), I believe it is best to apply that from the client. With pixels, images just look good in every format.

**Pull request merged**